hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/validate/UnsupportedOperationVisitor.java

```diff
@@ -194,6 +200,11 @@
         SUPPORTED_OPERATORS.add(HazelcastSqlOperatorTable.TO_TIMESTAMP_TZ);
         SUPPORTED_OPERATORS.add(HazelcastSqlOperatorTable.TO_EPOCH_MILLIS);
 
+        // JSON
+        SUPPORTED_OPERATORS.add(HazelcastSqlOperatorTable.JSON_QUERY);
+        SUPPORTED_OPERATORS.add(HazelcastSqlOperatorTable.JSON_PARSE);
```

Comment:
If we remove the JSON_PARSE function here, it will probably be unavailable to the user, but it will only be used internally. I wouldn't like to introduce a new non-standard function. It actually doesn't even parse the json, it just wraps it in `HazelcastJsonValue`, but no parsing is done.